src/storage/txn/scheduler.rs

```diff
@@ -958,6 +963,32 @@ fn process_write_impl(
     Ok(())
 }
 
+#[derive(Default)]
+struct ScheContext {
+    stats: HashMap<&'static str, StatisticsSummary>,
+}
+
+impl ScheContext {
+    fn add_statistics(&mut self, cmd_tag: &'static str, stat: &Statistics) {
+        let entry = self.stats.entry(cmd_tag).or_insert_with(Default::default);
+        entry.add_statistics(stat);
+    }
+}
+
+impl ThreadContext for ScheContext {
+    fn on_tick(&mut self) {
+        for (cmd, stat) in self.stats.drain() {
```

Comment:
too many for here, any way to optimize it? /cc @BusyJay

Reply:
There is not built-in cube support in iterator.